src/main/java/com/ceph/rados/IoCTX.java

```diff
@@ -685,4 +808,9 @@ public Integer call() throws Exception {
 
         return attr_map;
     }
+
+	@Override
+	public void close() throws IOException {
+        rados.rados_ioctx_destroy(getPointer());
+	}
```

Comment:
OK I did it.  I also fixed a few other minor places where tabs snuck in.

Lastly, I did a blanket reformat on the new Completion.java file to remove all the tabbing.  Some line breaks changed here and there, but there should not be any code changes.

Thanks!